sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+## Proposed Solution
+The sBTC protocol enables a secure transfer of Bitcoin to and from the Stacks blockchain, providing users with a digital asset backed 1:1 with BTC that can utilize the scalability and programmability of Stacks. Users can deposit BTC into the protocol and seamlessly transact using smart contracts on the Stacks blockchain, and have the freedom to redeem sBTC tokens for the underlying BTC at any time.
+
+The Stacks blockchain allows for onchain verification of the BTC held in the protocol, enhancing transparency compared to custodial approaches like WBTC. Additionally, the management of the Bitcoin script/wallet on the Bitcoin blockchain is decentralized, involving multiple participants rather than a single entity. This ensures a more resilient and trustworthy system, where signers are economically incentivized to execute peg-out transactions efficiently.
+
+Overall, the sBTC protocol not only addresses the limitations of the Bitcoin scripting system but also provides a secure and decentralized solution for utilizing Bitcoin in various applications.
+
+## Design
+To achieve these goals, this proposal implements the following features for sBTC:
+- sBTC is a SIP-10 token backed 1-1 by BTC.
+- The sBTC peg wallet is maintained by the set of sBTC signers. These signers are responsible for the security and maintenance of the wallet, ensuring that sBTC is redeemable for BTC.
+- To support deposits and withdrawals in sBTC, Signers must collectively make smart contract calls on Stacks. We require each signer to individually sign any contract call and ensure at least 70% of the signer signatures are present for each contract call to be valid.
+- Bitcoin can be converted into sBTC within 3 Bitcoin blocks. sBTC can be converted into Bitcoin within 6 Bitcoin blocks. This ensures sBTC on and off ramps are faster than other BTC assets on the market.
+- SIP-10 token contact remains consistent across sBTC releases. This provides reliability for users and developers, meaning no adjustment from builders will be needed as the system evolves through subsequent releases.
+
+### Iterating Toward an Open Signer Network
+This SIP describes an iterative approach to sBTC development via a release plan that is simpler to implement and helps accelerate the sBTC release timeline, but does not include the complete feature set described in the original sBTC design documents.
+
+Thus, the rollout for sBTC will include two key phases:
+
+**Bootstrap:** In this phase an initial group of Signers selected through an open community governance process will be responsible for Signing sBTC transactions on the network. The initial group of Signers will be unchanged throughout the Bootstrap phase of the release schedule. During this phase, sBTC will have a distinct signer set (known as the “sBTC Bootstrap Signer”). And sBTC will not explicitly be integrated into the Stacks Proof of Transfer consensus. As a result, this release can activate without a hard fork.
```

Comment:
Typo: "Signing" --> "signing"

Reply:
What are the candidacy criteria for the bootstrap signers?

Reply:
Ah, this is described later. Perhaps include a link, or simply say that it's provided below?

Reply:
I'm a little confused by this, along with the later text. 
you refer to an open community governance process for the bootstrap phase, with a distinct set of signers. does that mean anyone may be an sbtc-signer during the bootstrap phase?

my confusion comes from the next section, where you mention "decentralized Signer set is reached". 

reading between the lines, i think the bootstrap phase is a closed network of signers (initial partners etc), which is then opened after the bootstrap phase. if that's the case, i think some adjustments to the terminology are needed in this section.